risc0/zkvm/sdk/rust/methods/verify/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
```

Comment:
Can we just use the risc0-zkvm-methods-inner package?  This seems like a large amount of boilerplate just to add a single method.

Reply:
Yes, initially it had to be separate because I was building the receipt within the `build.rs`.

Reply:
I'm having trouble figuring out how to make `verify.rs` avoid `std` while also being in the same dir as the `Cargo.toml` that has `default = ["std"]`

Reply:
OK, I moved the verify package to be a sibling of inner, but I think they need to be separate due to differences in features and dependencies.